redbot/cogs/mod/mod.py

```diff
@@ -886,6 +886,7 @@ def __unload(self):
 
         perms_cache[str(channel.id)] = overwrites.send_messages
```

Comment:
Let's actually make this track both the original setting of the `send_messages` overwrite and the `add_reactions` overwrite so that we can just set it back to the old value on unmute